onnxruntime/core/optimizer/transformer_memcpy.cc

```diff
@@ -4,24 +4,29 @@
 #include "transformer_memcpy.h"
 #include "core/framework/kernel_registry_manager.h"
 #include "core/framework/execution_providers.h"
+#include "core/graph/graph_utils.h"
+#include "core/common/logging/logging.h"
+#include "core/common/logging/macros.h"
 
 using namespace ONNX_NAMESPACE;
+using namespace onnxruntime::logging;
+
 namespace onnxruntime {
 
 // implements MemCpy node insertion in graph transform
 // note that GraphTransformer::Apply() is supposed to be stateless, so this cannot derive from GraphTranformer
 class TransformerMemcpyImpl {
  public:
-  TransformerMemcpyImpl(onnxruntime::Graph& graph, const std::string& provider)
-      : graph_(graph), provider_(provider) {}
+  TransformerMemcpyImpl(onnxruntime::Graph& graph, const std::string& provider, const Logger& logger)
+      : graph_(graph), provider_(provider), logger_(logger) {}
```

Comment:
I don't see much value about adding logging info in this class.

Reply:
Removed all logging.